### PR TITLE
Sync files from gems dir as well

### PR DESCRIPTION
**Pull request by Sammers21**
In order to make gems available for download, we need to sync them with asto storage.

Related to #49.

### CODE REVIEW

File: src/main/ruby/SubmitGem.rb [hunk 1]
```diff
@@ -74,18 +74,29 @@ def sync(from, to)
...
-                   -> src, dest {
-                     src.removeAll(dest)
-                     src
-                   }
-               )
-               .flatMapPublisher { |keys| Flowable::from_iterable(keys) }
-    specs.mergeWith(diff).to_list.flatMapCompletable do |keys|
+    specs
+        .mergeWith(diff_keys("quick", from, to))
+        .mergeWith(diff_keys("gems", from, to))
```
Comment (g4s8) [inline src/main/ruby/SubmitGem.rb:79]: @Sammers21 it's not clear what is these magic `quicks` and `gems` directories, please add comments about it,

Comment (Sammers21) [reply src/main/ruby/SubmitGem.rb:79]: @g4s8, those are not magic, those name is a part of gem repository standard: S

> The actual .gem packages are found in the indexed gems directory. The RubyGems API expects direct access to download the installable gems via /gems path of your source:

Source: https://blog.packagecloud.io/eng/2015/12/15/rubygem-index-internals/

**Comment by 0crat** [comment]
Job was finished in 2 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)[/z](https://www.0crat.com/u/paulodamaso) didn't make CR comments

**Pull request merged**